hr_period/README.rst

```diff
@@ -0,0 +1,95 @@
+HR Payroll Period
+=================
+
+Add the concept of payroll period.
+
+The objective is not to restrict or complicate the payroll workflow.
+It is meant to reduce the number of clicks related to date selections and avoid mistakes.
+
+Add the date of payment on the payslip and payslip batch. This date is automatically filled when selecting
+a period.
+
+Add the company field on the payslip batch.
+Add a sequence on the payslip batch name.
+
+
+Installation
+============
+
+Install the payroll of your localisation, then install this module.
+
+
+Configuration
+=============
+
+Create a fiscal year
+--------------------
+Go to: Human Resources -> Configuration -> Payroll -> Payoll Fiscal Year
+
```

Comment:
I wouldn't call it "Payroll fiscal year", but "Payroll periods" or "Payroll intervals"